components/ScheduleHome.vue

```diff
@@ -1,15 +1,26 @@
 <template>
-  <div>
-    <div v-for="(day, index) in showsByDate" :key="day.id" class="mb-2 scheduleblock">
-      <div class="flex flex-row justify-between dayname">
-        <h4>{{ dayNames[day[0].day-1] }}</h4>
-        <span>{{ format(new Date(index === 0 ? todayDate :
-          tommorrow.setDate(tommorrow.getDate() + 1)) , 'MMMM dd.') }}</span>
+  <div class="h-full overflow-hidden bg-gray-300">
+    <div class="grid grid-cols-2 text-center schedule-head">
+      <div class="infos title">
+        <nuxt-link to="/shows">
+          <h3>Schedule</h3>
+        </nuxt-link>
       </div>
-      <div class="dayshows">
-        <div v-for="show in day" :key="show.id" :class="showAirCheck(index, show.name) ? 'dayblock onair' : 'dayblock'">
+      <div class="infos day">
+        <h3>{{ todayName }}</h3>
+      </div>
+    </div>
+    <div class="flex items-center scheduleblock">
+      <div class="w-full m-auto">
+        <div v-for="show in showsByDate[0]" :key="show.id" :class="showAirCheck(0, show.name) ? 'dayblock onair' : 'dayblock'">
```

Comment:
Maybe later on we should add a nice error message if there is no data?

Reply:
no data???

Reply:
Yeah maybe there's no feed coming from back-end or something. But it can be revisited later in focus